doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -25,3 +25,179 @@ All of the requirements for RC2 have been defined in the Reference Model (RM) an
+
+| Scenarios                                                          |
+|--------------------------------------------------------------------|
+| Kubernetes.create_and_delete_deployment                            |
+| Kubernetes.create_and_delete_job                                   |
+| Kubernetes.create_and_delete_namespace                             |
+| Kubernetes.create_and_delete_pod                                   |
+| Kubernetes.create_and_delete_pod_with_configmap_volume             |
+| Kubernetes.create_and_delete_pod_with_configmap_volume [2]         |
+| Kubernetes.create_and_delete_pod_with_emptydir_volume              |
+| Kubernetes.create_and_delete_pod_with_emptydir_volume [2]          |
+| Kubernetes.create_and_delete_pod_with_hostpath_volume              |
+| Kubernetes.create_and_delete_pod_with_secret_volume                |
+| Kubernetes.create_and_delete_pod_with_secret_volume [2]            |
+| Kubernetes.create_and_delete_replicaset                            |
+| Kubernetes.create_and_delete_replication_controller                |
+| Kubernetes.create_and_delete_statefulset                           |
+| Kubernetes.create_check_and_delete_pod_with_cluster_ip_service     |
+| Kubernetes.create_check_and_delete_pod_with_cluster_ip_service [2] |
+| Kubernetes.create_check_and_delete_pod_with_node_port_service      |
+| Kubernetes.create_rollout_and_delete_deployment                    |
+| Kubernetes.create_scale_and_delete_replicaset                      |
+| Kubernetes.create_scale_and_delete_replication_controller          |
+| Kubernetes.create_scale_and_delete_statefulset                     |
+| Kubernetes.list_namespaces                                         |
+
+The following software versions are considered to verify Kubernetes v1.18
+(latest stable release) selected by CNTT:
+
+| software                | version     |
+|-------------------------|-------------|
+| Functest                | kali        |
+| Kubernetes              | v1.18       |
+| xrally-kubernetes       | 1.1.1.dev12 |
+
+### Kubernetes API benchmarking
+
+[Rally](https://github.com/openstack/rally) and is tool and framework that
+allows to perform Kubernetes API benchmarking.
+
+[Functest Kubernetes Benchmarking](https://git.opnfv.org/functest-kubernetes/tree/docker/benchmarking/testcases.yaml?h=stable%2Fkali)
+proposed a Rally-based test case,
+[xrally_kubernetes_full](http://artifacts.opnfv.org/functest-kubernetes/LCY61RV15EX7/functest-kubernetes-opnfv-functest-kubernetes-benchmarking-kali-xrally_kubernetes_full-run-4/xrally_kubernetes_full/xrally_kubernetes_full.html),
+which iterates 10 times the mainline
+[xrally-kubernetes](https://github.com/xrally/xrally-kubernetes) scenarios.
+
+At the time of writing, no KPI is defined in
```

Comment:
What KPI is needed / expected? If known we can simply create a issue to get it added in. Is there an RA1 equivalent?

Reply:
It's the same for RA1. It would ask us to define all timeouts for the mainline Kubernetes (and OpenStack for RA1) operations. As far as I understand, the default values are correct and they are not overriden by any CNTT requirement.

It clarifies that we don't leverage on all Rally capabilities which allows very precise benchmarking.
For instance, you can find all durations calculated by Rally in the following link:
http://artifacts.opnfv.org/functest-kubernetes/HTJLR3AVHUN6/functest-kubernetes-opnfv-functest-kubernetes-benchmarking-kali-xrally_kubernetes_full-run-8/xrally_kubernetes_full/xrally_kubernetes_full.html#/Kubernetes.create_and_delete_deployment

Reply:
> As far as I understand, the default values are correct and they are not overriden by any CNTT requirement.

So, is all we need to do is create a requirement in RA2 that refers to the default timeouts?  If so, can you either create an issue accordingly and/or provide a link to the default values please?

Reply:
@tomkivlin Thank you for your quick feedbacks.
Please see https://github.com/cntt-n/CNTT/issues/1914 and https://github.com/cntt-n/CNTT/pull/1903
I'm waiting a few days for comment. For the being, I'm going to implement the third options as we both agree.